refinery/core/test_views.py

```diff
@@ -1089,3 +1092,47 @@ def test_get_event_list(self):
                 }
             ]
         )
+
+
+class CustomRegistrationViewTests(TestCase):
+    def test_user_registration_successful_recaptcha(self):
+        username = "new-test-user"
+        password = make_password('password')
+        response = self.client.post(
+            "/accounts/register/",
+            data={
+                "username": username,
+                "email": "test@example.com",
+                "first_name": "test",
+                "last_name": "user",
+                "affiliation": "Test Users",
+                "password1": password,
+                "password2": password
+            }
+        )
+        self.assertTrue(response.wsgi_request.recaptcha_is_valid)
```

Comment:
Why wsgi_request and not request?

Reply:
When we add the attribute `recapthca_is_valid` [here](https://github.com/refinery-platform/refinery-platform/pull/3006/files?utf8=%E2%9C%93&diff=unified&w=1#diff-f3493f14bf81ad37fdc507fe58c10e29R1011 ), we are adding it to the `WSGIRequest` instance. The Django test client only returns this through [`response.wsgi_request`](https://docs.djangoproject.com/en/1.8/topics/testing/tools/#django.test.Response.wsgi_request)